chia/full_node/coin_store.py

```diff
@@ -348,6 +348,98 @@ async def get_coin_states_by_puzzle_hashes(
 
         return coins
 
+    async def stream_coin_states_by_puzzle_hashes(
+        self,
+        puzzle_hashes: Set[bytes32],
+        min_height: uint32,
+        include_spent: bool,
+        include_unspent: bool,
+        include_hints: bool,
+        max_batch_size: int = 50000,
```

Comment:
when you have this many parameters of very similar type (especially the `bools`) I think it's really important to require these parameters be named. you can do that by adding `*` to the parameter list before the first names parameter. I would suggest before `min_height`